e_os.h

```diff
@@ -27,11 +27,8 @@
  * set this to a comma-separated list of 'random' device files to try out. By
  * default, we will try to read at least one of these files
  */
-#  if defined(__s390__)
-#   define DEVRANDOM "/dev/prandom","/dev/urandom","/dev/hwrng","/dev/random"
-#  else
-#   define DEVRANDOM "/dev/urandom","/dev/random","/dev/srandom"
-#  endif
+#  define DEVRANDOM "/dev/urandom", "/dev/random","/dev/hwrng", "/dev/srandom"
```

Comment:
Since you added spaces, you might as well add one before `"/dev/hwrng"` as well :wink:

Reply:
done.